woocommerce/woocommerce
# Fix date range and UI in Analytics

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This PR fixes a bug related to the UI and the date range picker under Analytics.

GB changed the dropdown/popover logic so it gets rendered by default outside of the element tree at the bottom of the page. This was causing a styling not to be applied, as it wasn’t part of that tree anymore.

This PR adds a couple of props to the component `date-range-filter-picker` to render it inline and not focus it on mount.

Closes #41297.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Checkout this branch and go to Analytics > Overview
2. Verify the Date range is looking correctly
3. Under the Date range component go to Custom and choose two dates
4. Everything should work well.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
